Suassuna-Dummy.pro

```diff
@@ -23,23 +23,61 @@ RCC_DIR = tmp/rc
 # Project libs
 LIBS *= -lprotobuf -lQt5Core
 
+# Compiling .proto files
+system(echo "Compiling protobuf files" && cd include/proto && protoc --cpp_out=../ *.proto && cd ../..)
```

Comment:
This is the command line used in .pro file to compile protobuf when you cast the **qmake** command to generate the **Makefile**